internal/k8s/entity.go

```diff
@@ -28,3 +29,47 @@ func ImmutableEntities(entities []K8sEntity) []K8sEntity {
 	}
 	return result
 }
+
+type LoadBalancer struct {
+	Name  string
+	Ports []int32
+}
+
+func ToLoadBalancers(entities []K8sEntity) []LoadBalancer {
+	result := make([]LoadBalancer, 0)
```

Comment:
what's the point of making a slice of size 0, and then put something in it? Shouldn't it be a slice of size 1?

Reply:
no, because that would create a slice with 1 element, the default element, which is not what we want.

Reply:
I see, so this could also be rephrased as:

```golang
result := []LoadBalancer{}
```